.github/labeler.yml

```diff
@@ -100,6 +100,12 @@
   - "**/*bluetooth*"
 "area: Bluetooth Mesh":
   - "subsys/bluetooth/mesh/**/*"
+"Bluetooth Controller":
```

Comment:
area: ...

Reply:
@nashif 
I figured that these should be 1:1 to the Github label names. 

The two new labels added by @carlescufi does not have the `area: ` prefix and are simply called `Bluetooth Host` and `Bluetooth Controller`.

Reply:
then when to fix those labels added by @carlescufi